modules/account-lib/test/unit/coin/algo/transactionBuilder/keyRegistrationBuilder.ts

```diff
@@ -110,6 +110,38 @@ describe('Algo KeyRegistration Builder', () => {
       should.deepEqual(txJson.genesisHash.toString('base64'), genesisHash);
     });
 
+    it('should build an online key registration transaction with stateProof Param', async () => {
```

Comment:
can we add another test where we try to add a stateProofKey that isn't in base64 format.

Reply:
Yes, that's a great test... I'm adding it.